width: 1280
height: 720
intrinsic_matrix:
  - 851.67697279
  - 0
  - 0
  - 0
  - 855.55430548
  - 0
  - 628.07270979
  - 354.57494198
  - 1
